rust/src/pgsql/pgsql.rs

```diff
@@ -632,10 +632,7 @@ unsafe extern "C" fn probing_parser_ts(
         let slice: &[u8] = build_slice!(input, input_len as usize);
 
         match parser::parse_request(slice) {
-            Ok((_, request)) => {
-                if let PgsqlFEMessage::UnknownMessageType(_) = request {
-                    return ALPROTO_FAILED;
-                }
+            Ok((_, _)) => {
```

Comment:
This does not look right to me : the code does not match the commit message : this is probing, not parsing

I do not think we want to detect pgsql when we see unknown messages